lib/vsc/utils/fancylogger.py

```diff
@@ -345,6 +345,8 @@ def deprecated(self, msg, cur_ver, max_ver, depth=2, exception=None, *args, **kw
             self.raiseException("DEPRECATED (since v%s) functionality used: %s" % (max_ver, msg), exception=exception)
         else:
             deprecation_msg = "Deprecated functionality, will no longer work in v%s: %s" % (max_ver, msg)
+            if verbose:
+                sys.stderr.write(deprecation_msg)
```

Comment:
deprecation warnings now only go to the log, which makes them practically invisible, you typically only dive into a log when something goes wrong

warnings should be printed when using deprecated functionality, like many other tools do too

if we don't want to do this in fancylogger, that's fine, then I'll construct some kind of wrapper around `log.deprecated` in EasyBuild